### PR TITLE
Added race history in spanish

### PR DESCRIPTION
**Pull request by nemux**
Please review this, @casey-collab

### CODE REVIEW

File: espanol/carrera/carrera.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Y así el fantástico piloto logro ganar la carrera...
```
Comment (casey-collab) [inline espanol/carrera/carrera.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by nemux** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**